generic3g/tests/Test_Scenarios.pf

```diff
@@ -329,6 +329,9 @@ contains
       expected_itemtype = get_expected_itemtype(expectations, _RC)
 
       itemtype = get_itemtype(state, short_name, _RC)
+      if (expected_itemtype /= itemtype) then
+         _HERE, msg
```

Comment:
In general, I would agree but:
1. This is inside a test
2. It only prints if an error has happened
3. I've had to add this print statement multiple times over the months.
The real fix is to improve the error handling so that the msg comes out with the actual `@assert_that()` just after.   And I'll do that in the next pass.  Leave it as it is for now.